AUTHORS

```diff
@@ -151,6 +149,7 @@ Theo Schlossnagle <jesus@omniti.com>
 Kai Chen <kaichenxyz@gmail.com>
 Daniel Chcouri <333222@gmail.com>
 Mihai Călin Bazon <mihai@bazon.net>
+Kevin Millikin <kmillikin@chromium.org>
```

Comment:
`@chromium.org` addresses should be filtered out.  Those come from cherry-picked V8 patches; their authors didn't contribute to node.js, they contributed to V8.